docs/kqueen.rst

```diff
@@ -102,7 +104,7 @@ Example configuration files are located in `config/` directory. Default configur
       - REST API for Jenkins
     * - JENKINS_PASSWORD
       - None
-      - Password for Jenkins login.
+      - Optional. Default password for Jenkins login. Is not used in UI
```

Comment:
Optional. Using in case of Queen-API standalone, otherwise, possibly to define Jenkins credentials from UI side

Reply:
See comment below, let's add Maria to check English grammer

Reply:
ok

Reply:
Optional. The default Jenkins password. Not used in the UI. 
Btw, is web UI meant here? If so, please change to "in the web UI."